source/_integrations/growatt_server.markdown

```diff
@@ -11,11 +11,15 @@ ha_iot_class: Cloud Polling
 
 This is a sensor to collect information from your Growatt inverters using [Growatt server](https://server.growatt.com/).
 
-This will log into your Growatt account and grab the first "Plant", after which it collects the inverters on this plant and creates sensors for these inverters as well as total sensors.
-
 ## Configuration
 
-Add the following to your `configuration.yaml` file:
+This integration supports being configured under the integrations button in home assistant.
+
+To configure it this way go to the configuration tab in your browser and click on integrations,
+Then click on the + and search for "Growatt Server".
+After which you can simply enter the required details.
```

Comment:
Maybe replace this with:

`To configure it this way go to the configuration tab in your browser and click on integrations. Next, click on the + and search for "Growatt Server" and enter the required details.`